inc/class-wpseo-utils.php

```diff
@@ -511,13 +511,13 @@ public static function clear_sitemap_cache( $types = array() ) {
 					$query .= ' OR ';
 				}
 
-				$query .= " option_name LIKE '_transient_wpseo_sitemap_cache_" . $sitemap_type . "_%' OR option_name LIKE '_transient_timeout_wpseo_sitemap_cache_" . $sitemap_type . "_%'";
+				$query .= " option_name LIKE '\_transient\_wpseo\_sitemap\_cache\_" . $wpdb->esc_like( $sitemap_type ) . "\_%' OR option_name LIKE '\_transient\_timeout\_wpseo\_sitemap\_cache\_" . $wpdb->esc_like( $sitemap_type ) . "\_%'";
```

Comment:
I think it's better to let WordPress do the escaping instead of partially escape the like part ourself. Just like:
```
$query .= " option_name LIKE '" . $wpdb->esc_like( '_transient_wpseo_sitemap_cache_' . $sitemap_type . '_' ) . "%' OR option_name LIKE '" . $wpdb->esc_like( '_transient_timeout_wpseo_sitemap_cache_' . $sitemap_type, '_' ) . "%'";
```